packages/document/main-doc/docs/en/blog/index.md

```diff
@@ -4,5 +4,70 @@ sidebar_position: 1
 
 # Blog
 
-You can find the latest news about Modern.js here.
+Find the latest update of modern.js and our thought processes here. Occasionally we explain the infrastructure behind it.
 
+---
+
+## Releasing MOdern.js v2: support Rspack
+
+> Published on 16.03.2023
+
+Hiyaaaaa~~ we are excited to announce the release of Modern.js v2!
+
+Modern.js is a suite of infrastructure tools we built for web development inside Bytedance (we call ourself Web-Infra-Dev internally...). Since we open sourced this project a little more than a year ago, there were dozens of contributors helped us on development; we aggregated more than 2,000 pull requests and modern.js started to support build tool like Rspack, nested routes, SSR(Streaming Server Render).
+
+We are extremely proud of what we have achieved so far, you can also refer to [this article](/blog/updates/v2-release-note)👈🏻 to see what has changed over the last year in modern.js.
+
+---
+
+## What is SSR(Streaming Server Rendering) in React 18
+
+> Published on 16.12.2022
+
+Since React 18, react supports a new type of SSR(streaming server rendering) and it brought two advantages over the react tool chain:
```

Comment:
"React" is also a proper noun.

```suggestion
Since React 18, React supports a new type of SSR(streaming server-side rendering) and it brought two advantages over the React tool chain:
```